src/colorscheme.jl

```diff
@@ -0,0 +1,81 @@
+const MONOKAY = Dict(
```

Comment:
Avoid copying the stuff from AutoViz in here it is going to be a nightmare to maintain!
There might be a way to support what you want within AutoViz